src/main/java/edu/ucdavis/dss/ipa/entities/InstructorType.java

```diff
@@ -0,0 +1,88 @@
+package edu.ucdavis.dss.ipa.entities;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import edu.ucdavis.dss.ipa.api.deserializers.InstructorTypeDeserializer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@SuppressWarnings("serial")
+@Entity
+@Table(name = "InstructorTypes")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@JsonDeserialize(using = InstructorTypeDeserializer.class)
+public class InstructorType extends BaseEntity {
+    private long id;
```

Comment:
This should be 'Long' instead of 'long', correct?

Reply:
Currently, all our entities use long (not Long) and the same transient pattern. I assume we were following tutorial code, but there may be a reason in some cases.

I would recommend we change them all together for consistency as a separate task so we can audit the use cases of the transient's that return 0 and ensure they are needlessly complex.